src/modules/tensor/cpu/kernel/jpeg_compression_distortion.cpp

```diff
@@ -86,24 +86,44 @@ alignas(32) const Rpp32f lumaQuantTable[64] = {
     72, 92, 95, 98, 112, 100, 103, 99
 };
 
+// Clamp a value between minVal and maxVal
+template <typename T>
+inline T clamp(T val, T minVal, T maxVal)
+{
+    return (val < minVal) ? minVal : (val > maxVal) ? maxVal : val;
```

Comment:
If this is the most optimal, this is a very generic function that need not be inside jpeg_compression_distortion.cpp specifically. Could be outside so other functions could use it. Can be changed later.